homeassistant/components/sensor/luftdaten.py

```diff
@@ -121,6 +126,9 @@ def device_state_attributes(self):
                 'lat': self.luftdaten.data.meta['latitude'],
                 'long': self.luftdaten.data.meta['longitude'],
             }
+            if self._show_on_map:
+                attr[ATTR_LATITUDE] = self.luftdaten.data.meta['latitude']
+                attr[ATTR_LONGITUDE] = self.luftdaten.data.meta['longitude']
```

Comment:
If `show_on_map` is `True` then we will have duplicate entries in the state card (lat/ and long/longitude). It should be either lat and long or latitude and longitude.

Reply:
OK - didn't know that that's a problem. I thought this way it will be always _non-breaking_.

Reply:
It's not a problem but looks wired and could confuse people because the same data is shown twice. .`[ATTR_LATITUDE]` and `[ATTR_LONGITUDE]` are exposed by default. Don't append it to the existing attributes, just make two different dict in regard of the configuration.

Reply:
ok, agreed! 👍